meta-mel/meta-altera/recipes-sato/webkit/webkitgtk_2.10.7.bb

```diff
@@ -76,3 +76,5 @@ FILES_${PN} += "${libdir}/webkit2gtk-4.0/injected-bundle/libwebkit2gtkinjectedbu
 
 # http://errors.yoctoproject.org/Errors/Details/20370/
 ARM_INSTRUCTION_SET = "arm"
+
+DEBUG_FLAGS_remove = "-g"
```

Comment:
@kergoth this only occurs when we build webkitgtk with Codebench, and it builds fine with internal toolchain.